nixos/modules/services/web-apps/shiori.nix

```diff
@@ -31,6 +31,20 @@ in {
         example = "/shiori";
         description = "The root of the Shiori web application";
       };
+
+      httpSecretKey = mkOption {
+        type = types.str;
+        default = "When empty all sessions will be invalidated on server restart";
```

Comment:
This seems like it should be the `description` and not the `default`?

Reply:
OOOOOOOOPS, fixed